lib/streaming-pipeline.js

```diff
@@ -222,6 +179,45 @@ module.exports = class StreamingPipeline extends Writable {
         }
     }
 
+    _wrapError(err) {
```

Comment:
this isn't wrapping the error as much as it's transforming the error to match grpc's expectations.

```suggestion
    _toGrpcError(err) {
```